script/utils/viemScriptHelpers.ts

```diff
@@ -1,12 +1,20 @@
 import { Chain, defineChain, getAddress } from 'viem'
 import networksConfig from '../../config/networks.json'
 import * as dotenv from 'dotenv'
+import consola from 'consola'
+import { getDeployments } from '../demoScripts/utils/demoScriptHelpers'
+import { SupportedChain } from '../demoScripts/utils/demoScriptChainConfig'
```

Comment:
Not important now but maybe should look into consolidating scripts/utils and shared items as something about importing types from a completely different scripts domain feels gross to me lol. Just a personal preference thing.

Reply:
yes we will consolidate this in a separate step